src/highlight.js

```diff
@@ -854,6 +854,7 @@ https://highlightjs.org/
 
     block.innerHTML = result.value;
     block.className = buildClassName(block.className, language, result.language);
+    block.setAttribute('data-language', block.getAttribute('data-language') || result.language);
```

Comment:
```suggestion
    block.dataset.language = block.getAttribute('data-language') || result.language;
```

Reply:
Ah I thought dataset was newer. Didn’t realize it was so old. Sure I can use newer api.

Reply:
Ah I was right. It’s not fully supported on older browsers. I know we can’t actively support all those older browsers but I’ve been trying not to PURPOSELY break them until 10.0. there are all sorts of little things to change once we jump over that fence. 

I’ll add this stuff to the 10.0 list.